crates/driver/src/tests/cases/limit_order/buy.rs

```diff
@@ -0,0 +1,235 @@
+//! Test that a buy limit order behaves as expected.
```

Comment:
Would be nice to qualify "behaves as expected", i.e. "what is being verified? and what is expected?"